lib/core/src/math/utils.rs

```diff
@@ -1,11 +1,11 @@
-#![allow(missing_docs)]
```

Comment:
Wait, I removed this file in the past. I don't think we should have `math` top-level package, as we do not have many utils there. Please keep them in sub-modules until we get significant amount of those.